### ISSUE TITLE
[C#] Getting extra models during client export

### ISSUE THREAD

**Issue by evenflow58** [swagger-api/swagger-codegen#6447]
Enter the following JSON into https://editor.swagger.io/ and generate a csharp client. In the models folder are Model.cs and Model1.cs. These look to be duplicates of MfdMemberViewModel but I'm not sure what is causing them to be created. Tests are also being created for these models.

Swagger-codegen version: 2.0
JSON gist: https://gist.github.com/evenflow58/801c6570ef36c1bdeafb958e48f9d5de

**Comment by kraihn** [comment]
I ran your spec against swagger-codegen version 2.2.2 and didn't get the Model.cs or Model1.cs files.

**Comment by evenflow58** [comment]
Yeah, I didn't have a problem with that either. Maybe the editor just hasn't been updated yet then. I'll just use the generator going forward. Thanks for the response.
